maven-plugins/enforcer-maven-plugin/src/main/java/io/helidon/build/maven/enforcer/nativeimage/NativeImageConfig.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2023 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.helidon.build.maven.enforcer.nativeimage;
+
+import java.util.List;
+
+import io.helidon.build.common.Lists;
+
+import org.apache.maven.plugins.annotations.Parameter;
+
+public class NativeImageConfig {
```

Comment:
class javadoc

Reply:
I am sorry about these javadoc, I ran once again `checkstyle` and `copyright` check locally and I got no warning. I am going to check manually that I did not forget anything.